modules/secrets-store-sync-secrets.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_content-type: PROCEDURE
+[id="secrets-store-sync-secrets_{context}"]
+= Enabling synchronization of mounted content as Kubernetes secrets
+
+You can enable synchronization to create Kubernetes secrets from the content on the mounted volume. An example where you might want to enable synchronization is to use an environment variable in your deployment to reference the Kubernetes secret.
+
+[WARNING]
+====
+Do not enable synchronization if you do not want to store your secrets on your {product-title} cluster and in etcd. Enable this functionality only if you require it; for example, to use environment variables to refer to the secret.
+====
+
+// TODO: If they enable sync, should we recommend they enable auto rotation too, or can they be done independent of each other?
+
+If synchronization is enabled, the secrets from the mounted volume are synchronized as Kubernetes secrets after you start a pod that mounts the secrets.
+
+The synchronized Kubernetes secret is deleted when all pods consuming the secret are deleted.
+
+// TODO: This came from upstream's: "When all the pods consuming the secret are deleted, the Kubernetes secret is also deleted."
+// What exactly does "consuming it" mean? Does it mean when the pod(s) that mounted the content are deleted, or does it also mean if there are pods out there that are referencing the Kubernetes secret via ENV VAR?
```

Comment:
@ropatil010 @knelasevero

Can either of you clarify what this means, about when the synced kube secret is deleted?

Reply:
@ropatil010 @knelasevero do either of you know what this was trying to say here? Or what we should say here about the sync'd secret getting deleted?

Reply:
> when the pod(s) that mounted the content are deleted

This :point_up: